Source/engine/render/scrollrt.cpp

```diff
@@ -1333,7 +1336,7 @@ void DrawFPS(const Surface &out)
 		framerate = 1000 * frameend / frames;
 		frameend = 0;
 	}
-	snprintf(string, 12, "%i FPS", framerate);
+	*fmt::format_to_n(string, 12, FMT_COMPILE("{} FPS"), framerate).out = '\0';

```

Comment:
While it's unlikely anyone is ever going to render a billion frames a second, this could potentially write past the end of the array? How about removing the `string` temporary and inlining the `fmt::format` call or using std::string instead of char[]?

Reply:
I'd like to minimize the per-frame allocations. Perhaps we can simply increase the buffer to the maximum number of digits.

Reply:
Done